src/components/playback/experimentalwarnings.js

```diff
@@ -46,21 +46,21 @@ define(['connectionManager', 'globalize', 'userSettings', 'apphost'], function (
     function showBlurayMessage() {
 
         var message =
-            'Playback of Bluray folders in this app is experimental. Some titles may not work at all. For a better experience, consider converting to mkv video files, or use an Jellyfin app with native Bluray folder support.';
+            'Playback of Bluray folders in this app is experimental. Jellyfin cannot decrypt DRM. The current behavior is to play all files in series, this can include a two minute black intro video.';
```

Comment:
I don't like `two minute long black video`. Also I'm not sure this is how it works, let's ask @Bond-009 as IIRC he was the last one to rework VOB support.